File: app/src/main/java/com/me/impression/db/model/HistoryRecord.java
package com.me.impression.db.model;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

/**
 * @author : Jie Cai
 * @desc : History Translate Record
 * */
@Entity
public class HistoryRecord {

    @PrimaryKey(autoGenerate = true)
    public Long id;

    public String from;

    public String to;

    public String srcText;

    public String destText;

    public String extra;

    public Long createTime;
}

File: app/src/main/java/com/me/impression/db/model/HistoryRecord.kt
package com.me.impression.db.model

import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * @author : Jie Cai
 * @desc : History Translate Record
 */
@Entity
data class HistoryRecord (
    @PrimaryKey(autoGenerate = true)
    var id: Long=0,
    var from: String="",
    var to: String="",
    var srcText: String="",
    var destText: String="",
    var extra: String="",
    var createTime: Long=0
)